run-nitro-enclaves-acm-tests

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+EXIT_FAIL=1
+
+# Binaries which should exist under $PATH after package installation
+binaries="nitro-cli \
+	vsock-proxy \
+	p11ne-agent \
+	p11ne-client"
+
+# Files brought in by the RPM(s), specified by full path
+files="/lib/systemd/system/nitro-enclaves-allocator.service \
+	/lib/systemd/system/nitro-enclaves-vsock-proxy.service \
+	/lib/systemd/system/nitro-enclaves-acm.service \
+	/usr/share/nitro_enclaves/p11ne/p11ne.eif \
+	/usr/share/nitro_enclaves/p11ne/image-measurements.json \
+	/usr/share/nitro_enclaves/p11ne/acm.example.yaml"
+
+
+function check_binaries() {
+	binaries="$1"
+
+	echo -n "Checking binaries exist in PATH .........."
+
+	for binary in $binaries; do
+		x="$(which $binary 2>&1)"
```

Comment:
`"$binary"`

Reply:
Fixed by making the `acm_ne_bins` an array of strings instead of a whitespace-separated string of binaries. Iterating through the array will make sure whitespace-separated binaries are getting the full name, not only the first token.